README.md

```diff
@@ -1 +1,60 @@
-EXPERIMENTAL WIP Json library based on a port of the [Jawn Parser](https://github.com/non/jawn) built on Arrow.
+# Helios
+
+Json library based on a port of the [Jawn Parser](https://github.com/non/jawn) built on Arrow.
+
+## How to use it
```

Comment:
Mention here that `jitpack` needs to be added

Reply:
this one https://github.com/jitpack/jitpack.io?